python/graphstorm/dataloading/dataloading.py

```diff
@@ -596,85 +611,91 @@ def node_feat_fields(self):
 
     @property
     def edge_feat_fields(self):
-        """ Edge features
+        """ Edge feature fields, which is given in class initialization.
 
         Returns
         -------
-        str or dict of list of str: Node feature fields in the graph.
+        str or dict of list of str: Edge feature fields in the graph.
         """
         return self._edge_feats
 
     @property
-    def pos_graph_feat_fields(self):
-        """ Get edge feature fields of positive graphs
+    def pos_graph_edge_feat_fields(self):
```

Comment:
This is a breaking change. Please highlight it in the PR description.

Reply:
Added in the PR description.